extensions/django_hooks/README.md

```diff
@@ -0,0 +1,7 @@
+# django_hooks
+
+Adds the ability to have hooks injected into templates, via the `hooks_tags.hook` template tag.
+
+Originally from https://github.com/nitely/django-hooks but after culling everything except the template hook.  The documentation on that template hook at [readthedocs](https://django-hooks.readthedocs.io/en/latest/hooks.html#templatehook) is [replicated locally](docs/templatehook.rst)
+
+The reason this was forked, rather than used as a module, is that the last commit from the other project was made in 2015, and can no longer be imported into a modern django project.  The template hook continues to work correctly, but the other hooks most likely do not, and so were removed.
```

Comment:
Can we move this information and the info in `docs/templatehook.rst` into hypha docs so we have it all in one place.

Reply:
Also we should have a place for documentation of added hooks.  Not sure where that should live.  I resurrected `docs/references/signals.md` in #4397, so maybe something like `docs/references/template_hooks.md`?

Reply:
I think `docs/references/template_hooks.md` sounds good. Easy to move if needed later on.